setup.py

```diff
@@ -1,15 +1,14 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from distutils.core import setup
-
-from setuptools import find_packages
+from setuptools import setup, find_packages
 
 from bio_hansel import __version__, program_name, program_desc
 
 classifiers = """
 Development Status :: 3 - Alpha
 Environment :: Console
-License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+License :: OSI Approved :: Apache Software License
```

Comment:
Why we change the license?

Reply:
I noticed that the license in the setup.py wasn't the same as the License file and readme, which both say it's Apache 2.0. I think we want the license to be Apache? We could check with everyone else about the proper license for biohansel.

Reply:
Good catch! Apache 2.0 is what other projects are.